lib/puppet.rb

```diff
@@ -211,7 +211,7 @@ def self.initialize_facts
   # code was deprecated in 2008, but this is still in heavy use.  I suppose
   # this can count as a soft deprecation for the next dev. --daniel 2011-04-12
   def self.newtype(name, options = {}, &block)
-    Puppet.deprecation_warning("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead.")
+    Puppet.deprecation_warning(_("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead."))
```

Comment:
Something that's become clear over the past few weeks as I've been working with translators on classifier service strings is that every time we have a variable we need to add a comment to translators to explain what the variable is a placeholder for, unless it is completely obvious. For example "Message: {1}" is obvious. @Magisus We may be able to get help with this as part of the string review. cc @kayayarai 

Having variables in strings is fine @MosesMendoza but I need to know the patterns we use so I can document them in [these translator instructions](https://docs.google.com/document/d/1NPa1uZYD8gqlMx7guuZlji6TheMfrQYRCjtJQxYg_10/edit?usp=sharing)